cluster/addons/rbac/node-proxy-role.yaml

```diff
@@ -10,15 +10,9 @@ rules:
   - ""
   resources:
   - nodes/proxy
-  verbs:
-  - create
-  - get
-- apiGroups:
-  - ""
-  resources:
   - nodes/log
   - nodes/stats
   - nodes/metrics
   - nodes/spec
```

Comment:
yeah, this is the role given to the credential the apiserver uses to reach the kubelet. some deployments use a superuser credential for that.